src/ir/value.h

```diff
@@ -50,8 +50,7 @@ class NamedValue {
   const T& element() const { return *element_; }
   absl::string_view name() const { return name_; }
   std::string ToString(SsaNames& ssa_names) const {
-    return absl::StrFormat("%%%d.%s", ssa_names.GetOrCreateID(*element_),
-                           name_);
+    return absl::StrFormat("%s", name_);
```

Comment:
Note that we are losing some critical information about the block or operation, which is needed. 

Why change this now? Can't we leave it as is? If you leave this unchanged, you don't have to change the tests in this PR also.

Reply:
`absl::StrFormat("%%%d.%s", ssa_names.GetOrCreateID(*element_), name_` this would cause parser to print out `%0.0 = core.minus [const: 10](arg0, arg1)\n"}))` instead of `%0 = core.minus [const: 10](arg0, arg1)\n"}))`. Would it make more sense to make this to keep ssa_names and remove name_. Like this `return absl::StrFormat("%%%d.%s", ssa_names.GetOrCreateID(*element_)`